malasakit-django/pcari/admin.py

```diff
@@ -146,7 +146,38 @@ def get_actions_wrapper(request):
 site.filter_actions(Group, ['delete_selected'])
 
 
-class ResponseAdmin(admin.ModelAdmin):
+class AdminViewMixin(admin.ModelAdmin):
+    """
+    Super class for admins to implement 'view' permissions.
+    """
+    def has_change_permission(self, request, obj=None):
```

Comment:
The name of this function does not match its specification.

Reply:
This is a default django function so I can't change its name. If this function returns false, the admin page will not show the model to the user.